common/browser.go

```diff
@@ -191,33 +186,39 @@ func (b *Browser) getPages() []*Page {
 }
 
 func (b *Browser) initEvents() error { //nolint:cyclop
-	var cancelCtx context.Context
-	cancelCtx, b.evCancelFn = context.WithCancel(b.ctx)
 	chHandler := make(chan Event)
 
-	b.conn.on(cancelCtx, []string{
+	// Using backgroundCtx here. Using vuCtx would close the connection/subprocess
```

Comment:
`backgroundCtx` is also here.

Reply:
Thanks for spotting this, updated in b52ef8a21fc0c009429b265b8b9c2b37c7a11cf5